policy/approval/evaluate.go

```diff
@@ -28,6 +28,13 @@ type evaluator struct {
 	root common.Evaluator
 }
 
+func (eval *evaluator) Trigger() common.Trigger {
+	if eval.root != nil {
+		return eval.root.Trigger()
+	}
+	return common.TriggerCommit
```

Comment:
```suggestion
	return common.TriggerStatic
```

If there's no root, there's nothing to evaluate.